style.css

```diff
@@ -467,20 +535,29 @@ footer {
   }
   to {
     width: 100%;
-    height: auto;
   }
 }
 
+@keyframes flush {
+  from {
+    border-right: transparent;
+  }
+  50% {
+    border-right: 2px solid var(--secondary);
+  }
+  to {
+    border-right: transparent;
+  }
+}
 
- /* Add animations */
-
```

Comment:
I could prefer not to name whatever you are doing, this makes your code not to be clean since it will have lots of naming.